test/util/all_close.hpp

```diff
@@ -50,19 +52,18 @@ namespace ngraph
                 {
                     if (count < 5)
                     {
-                        NGRAPH_INFO
-                            << std::setprecision(std::numeric_limits<long double>::digits10 + 1)
-                            << a[i] << " is not close to " << b[i] << " at index " << i;
+                        ar_fail << std::setprecision(std::numeric_limits<long double>::digits10 + 1)
+                                << a[i] << " is not close to " << b[i] << " at index " << i << "\n";
                     }
                     count++;
                     rc = false;
                 }
             }
             if (!rc)
             {
-                NGRAPH_INFO << "diff count: " << count << " out of " << a.size();
+                ar_fail << "diff count: " << count << " out of " << a.size() << "\n";
```

Comment:
Remove `if (!rc)` which is still using the old technique for limiting error reporting.